lib/clientsidescripts.js

```diff
@@ -676,20 +676,26 @@ functions.findByPartialButtonText = function(searchText, using) {
  * Find elements by css selector and textual content.
  *
  * @param {string} cssSelector The css selector to match.
- * @param {string} searchText The exact text to match.
+ * @param {string} searchText The exact text to match.  Can be a serialized regex.
  * @param {Element} using The scope of the search.
  *
  * @return {Array.<Element>} An array of matching elements.
  */
 functions.findByCssContainingText = function(cssSelector, searchText, using) {
   using = using || document;
 
+  if (searchText.indexOf('__REGEXP__') === 0) {
+    var match = searchText.split('__REGEXP__')[1].match(/\/(.*)\/(.*)?/);
+    searchText = new RegExp(match[1], match[2] || '');
+  }
   var elements = using.querySelectorAll(cssSelector);
   var matches = [];
   for (var i = 0; i < elements.length; ++i) {
     var element = elements[i];
     var elementText = element.textContent || element.innerText || '';
-    if (elementText.indexOf(searchText) > -1) {
+    
+    if ((searchText instanceof RegExp && searchText.test(elementText)) ||
+        elementText.indexOf(searchText) > -1) {
```

Comment:
My understanding is "elementText.indexOf(searchText) > -1" should only be applied to "searchText is not a RegExp"
For now, if elementText doesn't satisfy searchText regex, "elementText.indexOf(searchText) > -1" will still be called

Reply:
There are no side-effects to calling indexOf.  The way || works is that javascript will take the first true value.
So I don't think indexOf will be called.

Here is an example I tried out:
x = () => { console.log('here'); return true; }
y = () { console.log('there'); return false; }

z = x() || y()
here
true

Reply:
Ok, I tried to address this issue.  Let me know if you like the updated code.